app/views/wiki_pages/show.html.erb

```diff
@@ -2,4 +2,57 @@
   content_for :page_title, join_title(@page.title.to_s, @context.name)
   js_bundle :wiki_page_show
 %>
+
+<script>
```

Comment:
Let's just revert this change in the canvas code and commit it directly to the custom.js.  That file is in this repo: https://github.com/beyond-z/canvas-lms-js-css